helm/reana/values.yaml

```diff
@@ -113,3 +113,7 @@ traefik:
   ssl:
     enabled: true
     generateTLS: true
+
+# Quota
+quota:
```

Comment:
I see that,
1. Calculating the resource usage periodically
2. Calculating the resource usage always and always showing it

are two separate things in terms of configuration. While (1) is optional/configurable (2) should always be there even if the quotas have no limits, as I understood from Tibor explanations. (I am not sure whether we are having a naming/misundertanding issue implying that quotas have to always have limits, which if read by the dictionary it is true, but as we have been talking during the sprint we allow "infinite" limit to be a limit).

Regarding the shape of the config looks good, since it will allow us to grow in the direction of `quota.default_user_limit.cpu`, `quota.default_user_limit.disk` etc...